lib/portlayer/exec/commit.go

```diff
@@ -92,6 +92,12 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 		// clear the spec as we've acted on it - this prevents a reconfigure from occurring in follow-on
 		// processing
 		h.Spec = nil
+
+		// reconfigure vm display name to containerName-containerID
+		if err = c.UpdateDisplayName(ctx); err != nil {
+			return err
```

Comment:
It's not a right position to call rename. The container is already put into container cache, and container created event is published. Other work rely on this event will use wrong information.
Another question is about what happens if rename failed. Do you need to remove the vm previous created? If leave the vm here, and return error to persona, persona cache will not have this container, but portlayer cache will cache a container with wrong display name and half created configuration.